release/rke2/rke2.go

```diff
@@ -0,0 +1,85 @@
+package rke2
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v39/github"
+	ecmHTTP "github.com/rancher/ecm-distro-tools/http"
+	"github.com/rancher/ecm-distro-tools/repository"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	goDevURL           = "https://go.dev/"
+	goDevJSONPath      = "dl/?mode=json"
```

Comment:
It doesn't seem like the `goDevJSONPath` const needs to be a separate variable.

Reply:
Yes, I was testing something and forgot to change it back